.standard-GEM.md

```diff
@@ -1,110 +1,108 @@
 
 
 File tree
 ---------
 `/` signifies the root of the repository.  
 `.keep` files are used to indicate that the empty folder should not be ignored by _git_.
 
-- [x] `/.gitignore`  
+- [x] 🟥 `/.gitignore`  
 The repository must contain a `/.gitignore` file. This generic [.gitignore](https://git-scm.com/docs/gitignore) was prepared for multiple programming languages, and does not require modification.
 
-- [x] `/.github`  
+- [x] 🟥 `/.github`  
 The repository must contain a `/.github` folder, in which the contributing guidelines, code of conduct, issue templates and pull request templates must be placed. Defaults are provided and they do not require any modification.
 
-- [ ] `/.github/CONTRIBUTING.md`  
+- [ ] 🟥 `/.github/CONTRIBUTING.md`  
 This file is provided by the template, but it is empty. It must be filled in with the adequate contributing guideline instructions; a good example is https://github.com/SysBioChalmers/yeast-GEM/blob/master/.github/CONTRIBUTING.md.
 
-- [ ] `/code/README.md`  
+- [ ] 🟥 `/code/README.md`  
 The repository must contain a `/code` folder. This folder must contain all the code used in generating the model. It must also include a `README.md` file that describes how the folder is organized.
 
-- [ ] `/data/README.md`  
+- [ ] 🟥 `/data/README.md`  
 The repository must contain a `/data` folder. This folder contains the data used in generating the model. It must also include a `README.md` file that describes how the folder is organized.
 
-- [ ] `/modelFiles`  
+- [ ] 🟥 `/modelFiles`  
 The repository must contain `/modelFiles` folder.   
-This folder must contain the model files, in multiple formats, according to the guidelines below. As a general guideline, binary formats (`.mat`, `.xlsx`) must not exist on any other branches than _master_. Otherwise, with time, the size of the repository can create difficulties.  
+This folder must contain the model files, in multiple formats, according to the table below. As a general guideline, binary formats (`.mat`, `.xlsx`) must not exist on any other branches than _master_. The main reason for this is that binary files cannot be diff'ed, which means changes cannot be compared to previous versions, which introduces the chance of errors. Moreover, with time, the size of the repository can create difficulties, and we cannot yet recommend storing these files with Git LFS, as it introducs complexity.  
 All model files must be named the same as the repository, and with the appropriate extension.  
-Example: `yeast-GEM.mat`  
-| Model file format | _master_ branch | _devel_ and other branches |
-| ----------------- | --------------- | -------------------------- |
-| JSON `.json`      | must            | can                        |
+Example: `yeast-GEM.mat`
+
+| Model file format | _master_ branch | _develop_ and other branches |
+| ----------------- | --------------- | ---------------------------- |
+| JSON `.json`      | must            ||
```

Comment:
I sugget to modify the requirement for `JSON` model format in _`master`_ branch,  from "must" to "can". Because this probably has Not been implemented in any GEM-type repos yet, at lease in any Sysbio.

Reply:
You're right. I'm not sure why it was a `must` to begin with. Perhaps I got it confused with the model metadata, but that's something to address in an future version.